hw-model/src/openocd/openocd_server.rs

```diff
@@ -0,0 +1,228 @@
+// Licensed under the Apache-2.0 license
+//
+// Derived from OpenTitan's opentitanlib with original copyright:
+//
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::io::{BufRead, BufReader, Write};
+use std::net::TcpStream;
+use std::os::unix::process::CommandExt;
+use std::path::Path;
+use std::process::{Child, Command, Stdio};
+use std::time::{Duration, Instant};
+
+use anyhow::{bail, Context, Result};
+use once_cell::sync::Lazy;
```

Comment:
I think the standard library could be used here: https://doc.rust-lang.org/std/cell/struct.OnceCell.html

One less dependency :)

Reply:
oncecell is already brought into the workspace and used by another package: https://github.com/chipsalliance/caliptra-sw/blob/74ebff299062deac9d3bf152d5cb2997f6480e88/Cargo.toml#L161

I can explore cleaning up this and the other package up in a follow up.